testing/sdk_dcos.py

```diff
@@ -0,0 +1,28 @@
+'''Utilities relating to getting information about DC/OS itself
+
+************************************************************************
+FOR THE TIME BEING WHATEVER MODIFICATIONS ARE APPLIED TO THIS FILE
+SHOULD ALSO BE APPLIED TO sdk_dcos IN ANY OTHER PARTNER REPOS
+************************************************************************
+'''
+from enum import Enum
+
+import sdk_cmd
+
+
+class DCOS_SECURITY(Enum):
```

Comment:
Question: Does this add any advantage over `sdk_utils.is_strict()`? for the current usage?

Reply:
nope this is the same thing will replace it

Reply:
seems to be the same thing will tremove this file and use existing methods

Reply:
FYI : the reason sdk method takes from environment is because the endpoint `dcos-metadata/bootstrap-config.json` is relatively new. might worth adding a check on cluster version (there should be a helper to do this already) before assuming `mode = r['security']` works.

Reply:
since this is manually triggered, it may be easier to use the old method and just set the mode in the enviornment, although using this method makes it easier for the operator